integration/deploy/external_test.go

```diff
@@ -37,6 +37,7 @@ deploy:
     command: echo hola
 external:
   test:
+    icon: default
```

Comment:
Why do we need to set icon? If icon is not set we should be setting `default` as the default value